native_mate/native_mate/constructor.h

```diff
@@ -125,7 +125,7 @@ void InvokeNew(const base::Callback<Sig>& factory,
                v8::Isolate* isolate,
                Arguments* args) {
   if (!args->IsConstructCall()) {
-    args->ThrowError("Requires constructor call");
+    args->ThrowError("Constructor call is required.");
```

Comment:
The standard blink side error for this kind of thing is this

```
VM535:1 Uncaught TypeError: Failed to construct '{ConstructorName}': Please use the 'new' operator, this constructor cannot be called as a function
```

Can we use that?